gameplay/src/Container.cpp

```diff
@@ -67,6 +67,7 @@ Container::~Container()
     std::vector<Control*>::iterator it;
     for (it = _controls.begin(); it < _controls.end(); it++)
     {
+        (*it)->_parent = nullptr;
```

Comment:
What version of compilers are you targeting? nullptr has been around for quite a while.